src/main/groovy/com/pstehlik/groovy/gelf4j/appender/Gelf4JAppender.groovy

```diff
@@ -226,19 +174,23 @@ extends AppenderSkeleton {
 
   public void setMdcFields(List mdcFields) {
     this.mdcFields = mdcFields
+    this.mdcFields.each {
+      this.translatedMdcNames.put(it as String, 'mdc_' + ((it as String) =~ /([A-Z])/).replaceAll('_$1').toLowerCase())
```

Comment:
@michaelnielson-taulia The MDC property names are now translated on initialization and cached in a map, so there is no regex tax on logging anymore